uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Test/Fabricator/AbsenceType.php

```diff
@@ -20,6 +20,10 @@ public static function fabricate($params = []) {
       $params['title'] = static::nextSequentialTitle();
     }
 
+    if (empty($params['category'])) {
```

Comment:
When you try to create an Absence Type via the UI, this might also result in an error since the category field is now required. What would be the default for new Absence Types, Custom? 
We should make sure that the UI is not broken and also add a test to verify which category a new absence type will have. Also is there a plan to introduce this field in the Absence type creation UI?

Reply:
Shouldn't the validation to ensure that the category sent is part of the available category option values be added in this ticket? Or is there a separate task for that?

Reply:
There is a separate [ticket](https://compucorp.atlassian.net/browse/PCHR-4225) to handle all validation per category.